README.md

```diff
@@ -39,7 +39,7 @@ Install [Conda](https://docs.conda.io/en/latest/miniconda.html) if not already i
 To get started, use the `remotes` package to install Azure ML SDK for R from GitHub.
 
 ```R
-remotes::install_github('https://github.com/Azure/azureml-sdk-for-r')
+remotes::install_cran('azuremlsdk', repos = 'http://cran.us.r-project.org')

```

Comment:
I don't think we should change here as it's github readme.

Reply:
Should just show both methods?

Reply:
you're right. All the other R packages like Tensorflow also show only install_github in their README.